packages/core/src/Interweave.tsx

```diff
@@ -13,9 +13,10 @@ export default function Interweave(props: InterweaveProps) {
     matchers = [],
     onAfterParse = null,
     onBeforeParse = null,
-    tagName = 'span',
+    wrap = true,
     ...parserProps
   } = props;
+  const tagName = parserProps.tagName || 'div';
```

Comment:
Should stay `span` otherwise its a breaking change.

Reply:
Then it should be documented. https://milesj.gitbook.io/interweave/ says under *Markup & Interweave props*:

> tagName (string | 'fragment') - The HTML element tag name to wrap the output with. Defaults to "div".

I will remove this commit.